core/arch/arm/plat-synquacer/conf.mk

```diff
@@ -0,0 +1,28 @@
+PLATFORM_FLAVOR ?= developerbox
+
+$(call force,CFG_GENERIC_BOOT,y)
+$(call force,CFG_SECURE_TIME_SOURCE_CNTPCT,y)
+$(call force,CFG_GIC,y)
+$(call force,CFG_PL011,y)
+$(call force,CFG_PM_STUBS,y)
+
+ifeq ($(PLATFORM_FLAVOR),developerbox)
+include core/arch/arm/cpu/cortex-armv8-0.mk
+CFG_TEE_CORE_NB_CORE = 24
+CFG_TZDRAM_START ?= 0xfc000000
+CFG_TZDRAM_SIZE ?= 0x03c00000
+CFG_SHMEM_START ?= 0xffc00000
+CFG_SHMEM_SIZE ?= 0x00400000
+endif
+
+ifeq ($(platform-flavor-armv8),1)
+$(call force,CFG_WITH_ARM_TRUSTED_FW,y)
+endif
+
+$(call force,CFG_WITH_LPAE,y)
+ta-targets = ta_arm64
```

Comment:
I'm puzzled here. You're not adding `ta_arm32` to `ta-targets` so this means only 64-bit TAs are supported. Which implies a 64-bit TEE core (kernel), too. But I don't see `CFG_ARM64_core` being set to `y`. 

In order to support "the usual mix" (32-bit kernel with 32-bit TAs, and 64-bit kernel with 32-bit or 64-bit TAs), you would need something like that: https://github.com/OP-TEE/optee_os/blob/master/core/arch/arm/plat-hikey/conf.mk#L13-L20

Reply:
I did pass CFG_ARM64_core=y from cmdline. Let me add $(call force,CFG_ARM64_core,y) in this file instead. For now I target only 64-bit TAs. In future, if we see any requirement for 32-bit TAs, we can add corresponding target.

Reply:
Fine with me.